isb_lib/sesar_adapter.py

```diff
@@ -20,7 +20,11 @@
 import isamples_metadata.SESARTransformer
 
 HTTP_TIMEOUT = 10.0  # seconds
+##############
+# Per Lulin, we shouldn't be using this as they are wanting to deprecate it
```

Comment:
Unrelated comment, but I was making changes and wanted to make sure the code was documented that we shouldn't be using this anymore.